src/languages/mathematica.js

```diff
@@ -1,59 +1,130 @@
+    disableAutodetect: true,
+    theming: {
+      aliases: {
+        brace: 'punctuation',
+        pattern: 'type',
+        slot: 'type',
+        symbol: 'variable',
+        'named-character': 'variable',
+        'builtin-symbol': 'keyword',
+        'message-name': 'string'
+      }
     },
     contains: [
-      hljs.COMMENT('\\(\\*', '\\*\\)', {contains: ['self']}),
+      PATTERNS,
+      SLOTS,
+      MESSAGES,
+      SYMBOLS,
+      NAMED_CHARACTER,
       hljs.QUOTE_STRING_MODE,
-      hljs.C_NUMBER_MODE
+      hljs.COMMENT(/\(\*/, /\*\)/, {contains: ['self']}),
```

Comment:
Any reason you moved comment so far down?  Usually it comes first so no other things get "stuck" inside comments...

Reply:
I think we can move the comment to the top, but I'd like to test this first. But the `(*` pattern in unique for comments as far as I know.